### DISCUSSION TITLE
webpage shows This Machinaris worker is controlled at: http://:8926

### DISCUSSION THREAD

**Discussion by steamch** [guydavis/machinaris#318] [Q&A] [answered]
System setup:

OS: UNRAID 6.9.2
Docker version number : 20.10.5
Machinaris branch: Main
Machinaris version number: 0.5.7
[https://imgur.com/Gbq1D4F](url)

**Comment by guydavis** [comment, marked as answer]
Hi sorry you have encountered this.  I think you're seeing the same issue related to Unraid CA templates as [here](https://forums.unraid.net/topic/108896-support-machinaris-chia-cryptocurrency-farming-plotman-plotting-unraid-webui/?do=findComment&comment=1041641).
My suggested solution is [here](https://forums.unraid.net/topic/108896-support-machinaris-chia-cryptocurrency-farming-plotman-plotting-unraid-webui/?do=findComment&comment=1041706).  Hope this helps!  If so, please mark the Dicussion as Answered.  If not, please reply with additional details.

**Reply by steamch** [reply]
> Hi sorry you have encountered this. I think you're seeing the same issue related to Unraid CA templates as [here](https://forums.unraid.net/topic/108896-support-machinaris-chia-cryptocurrency-farming-plotman-plotting-unraid-webui/?do=findComment&comment=1041641). My suggested solution is [here](https://forums.unraid.net/topic/108896-support-machinaris-chia-cryptocurrency-farming-plotman-plotting-unraid-webui/?do=findComment&comment=1041706). Hope this helps! If so, please mark the Dicussion as Answered. If not, please reply with additional details.

thank you
